ketcher-autotests/tests/File-Management/Open-And-Save-Files/MOL-Files/mol-files.spec.ts

```diff
@@ -547,3 +549,268 @@ test('Open V3000 file with R-Groups with Fragments', async ({ page }) => {
+        pathToOpen: 'Molfiles-V3000/multi-V3000.mol',
+        pathToExpected: 'Molfiles-V3000/multi-V3000-expected.mol',
+        format: 'v3000',
+      },
+      {
+        testName:
+          'Open and Save file - Open/Save  V3000 mol file contains Sgroup 2/2 - save',
+        pathToOpen: 'Molfiles-V3000/sgroup-different-V3000.mol',
+        pathToExpected: 'Molfiles-V2000/sgroup-different-V2000-expected.mol',
+        format: 'v2000',
+      },
+      {
+        testName:
+          'Open and Save file - Open/Save v3000 mol file with assigned Alias 2/2 - save',
+        pathToOpen: 'Molfiles-V3000/mol-12963-to-open.mol',
+        pathToExpected: 'Molfiles-V3000/mol-12963-to-open-expected.mol',
+        format: 'v3000',
+      },
+    ];
+
+    for (const file of files) {
+      test(`${file.testName}`, async ({ page }) => {
+        await openFileAndAddToCanvas(file.pathToOpen, page);
+
+        const expectedFile = await getMolfile(
+          page,
+          file.format as MolfileFormat,
+        );
+        await saveToFile(file.pathToExpected, expectedFile);
+
+        const METADATA_STRING_INDEX = [1];
+        const { fileExpected: molFileExpected, file: molFile } =
+          await receiveFileComparisonData({
+            page,
+            expectedFileName: `tests/test-data/${file.pathToExpected}`,
+            fileFormat: file.format as MolfileFormat,
+            metaDataIndexes: METADATA_STRING_INDEX,
+          });
+
+        expect(molFile).toEqual(molFileExpected);
+      });
+    }
+  });
+
+  test.skip('Open and Save files - V3000 mol file contains different Bond properties', async ({
+    page,
+  }) => {
+    /**
+     * Test case: EPMLSOPKET-1853
+     * Description: Structre is correctly generated from Molstring and vise versa molstring is correctly generated from structure.
+     * A file with V3000 format is resaved in V2000 format
+     *
+     * Now we can`t open the file - `Convert error! Cannot deserialize input JSON.`
+     * https://github.com/epam/ketcher/issues/2378
+     */
+
+    await openFileAndAddToCanvas(
+      'Molfiles-V3000/marvin-bond-properties-V3000(1).mol',
+      page,
+    );
+
+    const expectedFile = await getMolfile(page, 'v2000');
+    await saveToFile(
+      'Molfiles-V2000/marvin-bond-properties-V3000-expected.mol',
+      expectedFile,
+    );
+
+    const METADATA_STRING_INDEX = [1];
+    const { fileExpected: molFileExpected, file: molFile } =
+      await receiveFileComparisonData({
+        page,
+        expectedFileName:
+          'tests/test-data/Molfiles-V2000/marvin-bond-properties-V3000-expected.mol',
+        fileFormat: 'v2000',
+        metaDataIndexes: METADATA_STRING_INDEX,
+      });
+
+    expect(molFile).toEqual(molFileExpected);
+  });
+
+  for (let i = 1; i < 9; i++) {
+    test(`'Open and Save file - Open/Save files for ferrocen-like structures 1/2 - open ferrocene_radical0${i}.mol`, async ({
```

Comment:
guess you can remove `'` in test name :)

Reply:
Thank you, fixed